core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -143,21 +143,21 @@ class AdminClient(val time: Time,
                              clientHost: String,
                              assignment: List[TopicPartition])
 
-  def describeConsumerGroup(groupId: String): List[ConsumerSummary] = {
+  def describeConsumerGroup(groupId: String): Option[List[ConsumerSummary]] = {
```

Comment:
It's probably fine as it is. I was just thinking it's kind of a shame to throw away information such as the state and the members of the group, which may be helpful in some situations. But since `AdminClient` is not a public API, we can always change it later if we need to.